base/multi.jl

```diff
@@ -2121,9 +2129,10 @@ completion. To wait for completion, prefix the call with [`@sync`](@ref), like :
 """
 macro parallel(args...)
     na = length(args)
-    if na==1
+    if na == 1
         loop = args[1]
-    elseif na==2
+    elseif na == 2
+        depwarn("@parallel with a reducer is deprecated. Use ParallelAccumulators for reduction.", :@parallel)
```

Comment:
it'll be a conflict magnet so it can wait a bit, but whenever this proceeds, would be good to leave a comment in the appropriate section of deprecated.jl as a reminder to remove this code